force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Salesforce Flows: Lookup without null check
#### Severity
Warning

#### Finding
Flow "Error\_Handler" performs a record lookup without checking for null or empty values

#### Why is this a problem?
Flows that perform record lookups should make sure that the records are found. Not performing such a check may cause the Flow to break.

#### Help and documentation

* [Get Started With Flows (Trailhead)](https://trailhead.salesforce.com/en/content/learn/modules/flow-basics/get-started-with-flows) 
* [Flow Best Practices (Developer Documentation)](https://help.salesforce.com/s/articleView?id=sf.flow\_prep\_bestpractices.htm&type=5)